modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -3,32 +3,34 @@
 // storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="lvms-upgrading-lvms-on-sno_{context}"]
-= Upgrading {lvms} on {sno} clusters
+[id="lvms-updating-lvms_{context}"]
+= Updating {lvms} on a single-node OpenShift cluster
 
-You can upgrade the {lvms-first} Operator to ensure compatibility with your {sno} version.
+You can update {lvms} to ensure compatibility with the {sno} version.
 
 .Prerequisites
 
-* You have upgraded your {sno} cluster.
+* You have updated your {sno} cluster.
 
-* You have installed a previous version of the {lvms} Operator.
+* You have installed a previous version of {lvms}.
 
 * You have installed the OpenShift CLI (`oc`).
```

Comment:
There is an attribute for "OpenShift CLI (`oc`)" now if you want it
```suggestion
* You have installed the {oc-first}.
```

Reply:
Updated as per the suggestion.